webapp/pom.xml

```diff
@@ -121,6 +121,26 @@
             <artifactId>atlas-client-v2</artifactId>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.jsoup/jsoup -->
```

Comment:
Do we need both jsoup and owasp here?

Reply:
Oh mistake from my part, need to remove those.